packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,24 @@ description: A common platform interface for the url_launcher plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.8
+version: 2.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
-  plugin_platform_interface: ^1.0.1
+  # TODO (mvanbeusekom): use pub.dev once 1.10.0-nullsafety released.
+  plugin_platform_interface:
```

Comment:
It looks like this would need to be a source dependency https://api.cirrus-ci.com/v1/task/4983717032624128/logs/main.log